addOns/automation/src/main/java/org/zaproxy/addon/automation/VerificationData.java

```diff
@@ -231,7 +236,12 @@ public void initAuthenticationVerification(Context context, AutomationProgress p
                             this.getLoggedOutRegex()));
         }
         authMethod.setPollUrl(this.getPollUrl());
-        authMethod.setPollFrequency(JobUtils.unBox(this.getPollFrequency()));
+        if (getPollFrequency() != null) {
+            authMethod.setPollFrequency(getPollFrequency());
+        } else if (getPollFrequency() == 0) {
```

Comment:
We'll never go down this path, will we? if it return `0` then we'll go down the path above...

Reply:
I changed it :wink: